api/tests/Controller/ProfileControllerTest.php

```diff
@@ -7,14 +7,16 @@
 use ApiPlatform\Symfony\Bundle\Test\ApiTestCase;
 use ApiPlatform\Symfony\Bundle\Test\Client;
 use App\Controller\ProfileController;
-use App\Tests\Api\RefreshDatabaseTrait;
+use App\Tests\Fixtures\Story\DefaultUsersStory;
+use Zenstruck\Foundry\Test\Factories;
+use Zenstruck\Foundry\Test\ResetDatabase;
 
 /**
  * @see ProfileController
  */
 final class ProfileControllerTest extends ApiTestCase
 {
-    use RefreshDatabaseTrait;
+    use ResetDatabase, Factories;
```

Comment:
PSR-12 recommends to include one-per-line for use traits: https://www.php-fig.org/psr/psr-12/#42-using-traits